packages/code-studio/src/styleguide/TimeSliderInputs.tsx

```diff
@@ -41,10 +47,16 @@ class TimeSliderInputs extends PureComponent<
         <TimeSlider
           startTime={startTime}
           endTime={endTime}
+          isStartModified={isStartModified}
+          isEndModified={isEndModified}
           onChange={this.handleSliderChange}
         />
-        <p>StartTime: {TimeUtils.formatTime(startTime)}</p>
-        <p>EndTime: {TimeUtils.formatTime(endTime)}</p>
+        <p style={isStartModified ? { color: 'var(--dh-color-modified)' } : {}}>
```

Comment:
Should we be using style tag directly here or a CSS class name and put this in the CSS?